src/protocol/resp/src/message/bulk_string.rs

```diff
@@ -18,9 +18,14 @@ impl BulkString {
             inner: Some(bytes.into()),
         }
     }
+
     pub fn len(&self) -> usize {
         self.inner.as_ref().map(|i| i.len()).unwrap_or(0)
     }
+
+    pub fn nil() -> Self {
+        Self { inner: None }
```

Comment:
nit: add `$-1\r\n` in a comment as the serialized version

Similar symmetric comment to line 60/61.